CHANGELOG.md

```diff
@@ -651,12 +651,12 @@
 <a name="1_5_0_drop_node_10_support"></a>
 - [[repo]](#1_5_0_drop_node_10_support) support for `Node 10` is dropped. [#8290](https://github.com/eclipse-theia/theia/pull/8290)
   - From now on, Node.js `12.x` is required when building.\
-   The recommended minimum version is aligned with `electron` (Node.js `12.14.1`).
+   The recommended minimum version is aligned with `electron` (Node.js `12.18.3`).
```

Comment:
You shouldn't update old changelogs. Instead, add a breaking change section to version `1.17.0`